src/Pages/ProductDetails/ComponentYH/RightSlider.scss

```
@@ -0,0 +1,171 @@
+.RightSlider {
+    padding: 8.33vw 12.5vw 0px;
+    width: 50vw;
+    height: 80vw;
+    
+    .ProductName {
+        h1{
+            text-transform: uppercase;
+            font-size: 3.13vw;
+            font-weight: bold;
+        }
+
+        span {
+            text-transform: uppercase;
+            font-weight: bold;
+            font-size: 1.74vw;
+        }
+    }
```

공통으로 적용할 속성이라면 상위에서 한번만 주면 되겠죠? 
```suggestion
    .ProductName {
            text-transform: uppercase;
            font-weight: bold;
            
            h1 {
                font-size: 3.13vw;
            }
            
            span {
                font-size:1.74vw;
            }
}
```